Sources/Shapes.swift

```diff
@@ -43,9 +43,11 @@ public extension Path {
     static func ellipse(width: Double, height: Double, segments: Int = 16) -> Path {
         var points = [PathPoint]()
         let segments = max(3, segments)
-        let step = 2 * Double.pi / Double(segments)
-        let w = max(abs(width / 2), epsilon), h = max(abs(height / 2), epsilon)
-        for angle in stride(from: 0, through: 2 * .pi + epsilon, by: step) {
+        let step = Angle.twoPi / Double(segments)
+        let w = max(abs(width / 2), epsilon)
+        let h = max(abs(height / 2), epsilon)
+        for radians in stride(from: 0, through: Angle.twoPi.radians + epsilon, by: step.radians) {
+            let angle = Angle.radians(radians)
```

Comment:
Using Angle for internal APIs seems like it adda a lot of code, and I'm not sure what value it brings. The main benefit of Angle is to disambiguate whether an API parameter is in degrees or radians - it doesn't seem very relevant to code like this.

Reply:
Ok, fair enough.